wellregistry/registry/tests/admin/test_monitoring_location.py

```diff
@@ -150,6 +153,13 @@ def test_changelist_view_with_usgs_user(self):
         self.assertNotIn(b'Add monitoring location', resp.content)
         self.assertNotIn(b'Bulk Upload', resp.content)
 
+    def test_site_no_auto_complete_View(self):
+        client = Client()
+        client.force_login(self.usgs_user)
+        resp = client.get('/registry/admin/registry/monitoringlocation/siteno/autocomplete/?site_no__exact=12345678')
+
+        self.assertIn(b'12345678', resp.content)
+
```

Comment:
I would like to see a test where the site_no does not have a match (the results in the response is empty).

Reply:
added